src/net/sourceforge/kolmafia/request/FightRequest.java

```diff
@@ -10468,6 +10468,18 @@ else if (responseText.contains("Rock Promoters are long gone")) {
             QuestManager.updateCyrusAdjective(itemId2);
           }
           break;
+        case ItemPool.SHADOW_BRICK:
+          if (responseText.contains("They collide, and are annihilated")) {
+            itemSuccess = true;
+          }
+      }
+    }
+
+    if (itemSuccess || itemRunawaySuccess) {
+      var limit = DailyLimitType.USE.getDailyLimit(itemId);
```

Comment:
the only thing using this currently is the shadow brick, is that right?

Reply:
correct!